src/styles/main.scss

```diff
@@ -1,3 +1,62 @@
+$space: 1px;
+$size__box-day: 100px;
+$width-calendar: $size__box-day * 7 + $space * 6;
+$days: "mon" 0 , "tue" 1 , "wed" 2 , "thu" 3 , "fri" 4 ,"sat" 5 , "sun" 6;
+$first-day-margin: $size__box-day + $space;
+
 body {
   margin: 0;
+  min-height: 100vh;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.calendar {
```

Comment:
Create a separate file for calendar BEM block
![image](https://github.com/mate-academy/layout_calendar/assets/96409563/22241188-4a7c-4f60-9a45-1092c67368c1)